modules/music.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'item'
+require_relative 'genre'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, on_spotify)
+    super(publish_date)
+    @on_spotify = on_spotify
+  end
+
+  private
+
+  def can_be_archived?
+    super && on_spotify == true
+  end
+end
+
+# ms = MusicAlbum.new(true, 'rock','2020-04-03')
+# print ms.genre
```

Comment:
- [ ] Kindly remove these commented codes here to achieve a more clean codebase

Reply:
@vic778 and @chukwuemeka1234, I noticed you deleted one of my review comments and I believe it might be by mistake because You implemented all the changes I requested and that's why I have approved your PR. Kindly do not make this mistake again okay 😄 . Thank you.